File: Bahasa c/PRAK305-2210817220030-Valencia Angelin Nata.c
#include <stdio.h>
int main(){
int bd,m,j,s,h;
scanf("%d", &bd);
h= (bd%(86400*30))/86400;
j= (bd%86400)/3600;
m= (bd%3600)/60;
s= (bd%60);
if (h==0){
    printf ("%02d:%02d:%02d\n\n",j,m,s);
    }
else{
    printf("%d hari %02d:%02d:%02d\n\n",h,j,m,s);
    }
}

File: Bahasa python/PRAK305-2210817220030-Valencia Angelin Nata.py
bd= int (input())
h= (d%(86400*30))/86400
j= (d%86400)/3600
m= (d%3600)/60
s= (d%60)
if (h>=1):
    print ("%d hari %02d:%02d:%02d\n"%(h,j,m,s))
else:
    print("%02d:%02d:%02d\n"%(j,m,s))